Entities/Characters/Builder/BuilderCommon.as

```diff
@@ -32,22 +32,20 @@ Vec2f getBuildingOffsetPos(CBlob@ blob, CMap@ map, Vec2f required_tile_space)
 	return alignedWorldPos;
 }
 
-CBlob@ server_BuildBlob(CBlob@ this, BuildBlock[]@ blocks, uint index)
+CBlob@ server_BuildBlob(CBlob@ this, BuildBlock[]@ blocks, const u32 &in index)
```

Comment:
Is there a reason for this change?

Reply:
Honestly looks like that got in because of my habit. It wont do anything to hurt though